### PR TITLE
Add MinPubSub@0.0.2 w/ npm auto-update

### LINKED ISSUE #8658: [Request] Add MinPubSub
**Library name:** MinPubSub 
**Git repository url:**https://github.com/daniellmb/MinPubSub
**npm package url(optional):** 
**License(s):**MIT
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #8658
Related issue(s): #9414

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/daniellmb/MinPubSub
 * NPM package url (optional): https://www.npmjs.com/package/minpubsub
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 201
   - Count of watchers: 9
   - Count of forks: 45
 * NPM download stats (optional): 
   - Downloads in the last day: 2
   - Downloads in the last week: 9
   - Downloads in the last month: 31

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8658, cc #9414 @daniellmb

### CODE REVIEW

File: ajax/libs/MinPubSub/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "license": "MIT",
+  "npmName": "minpubsub",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "minpubsub*"
+      ]
+    }
+  ],
```
Comment (PeterDaveHello) [inline ajax/libs/MinPubSub/package.json]: @clairetsai818 we should remove the keys with leading underscore

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 I think you should
1) tell me what have you done
2) wait for the CI 
before you tag me to review it

File: ajax/libs/MinPubSub/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "publish",
+    "subscribe",
+    "pubsub",
+    "observer"
+  ],
+  "author": {
+    "name": "Daniel Lamb",
+    "email": "daniellmb.com"
+  },
+  "dependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/MinPubSub/package.json]: Please remove empty section.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've removed empty section. Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 please don't forget to rebase your branch with the latest master branch next time when you need to update the PR.

**Pull request merged**